tools/update_runnablejobs.py

```diff
@@ -55,11 +55,16 @@ def update_runnableapi():
 
 def query_the_runnablejobs(new_timestamp, task_id=None):
     if task_id:
-        url = TREEHERDER_HOST.format('mozilla-inbound', task_id)
-        data = retry(requests.get, args=(url, ), kwargs={'headers': headers}).json()
-        if len(data['results']) > 0:
-            data['meta'].update({'timetamp': new_timestamp})
-            return data
+        try:
+            url = TREEHERDER_HOST.format('mozilla-inbound', task_id)
+            data = retry(requests.get, args=(url, ), kwargs={'headers': headers}).json()
+            if len(data['results']) > 0:
+                data['meta'].update({'timetamp': new_timestamp})
+                return data
```

Comment:
here you return the data, but in the except clause you do not, please move the return data line outside of the try/except block.